### PR TITLE
[New] add mint-ui v2.2.7 w/ npm auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
@Amomo @Piicksarn could u help me check this PR for #9357 ? thank you!
since this lib is a little big, I explain my step of this PR:

> pnpm i mint-ui
> copy files under `/lib` to make 2.0.2
> minify all folder under `2.0.2` 
> add some infos to package.json 

**git repo url:** https://github.com/ElemeFE/mint-ui
**Watch 146
  Star 2,838
 Fork 379**

**npm package url:** https://www.npmjs.com/package/mint-ui
**216 downloads in the last day
667 downloads in the last week
2,631 downloads in the last month**

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR 
- [x] The lib has notable popularity  
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [x] More than 500 downloads stats per month on npm registry 
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [ ] Has valid tags for each versions (for git auto-update) 
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Ping @Piicksarn

**Comment by Piicksarn** [comment]
@cdnjs/intern @cdnjs/intern2, excuse me!
I have a question is about :
`remove *.map to match the file structure of file had been minified by the author`
Should we always need to remove *.map to match the file structure of library?

**Comment by PeterDaveHello** [comment]
That's not right at all, we've discussed about this before, the removal of map files will only make more problems, the right way is to not generate the map file. The structure is important so we need to keep it at least for the old versions.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by maruilian11** [comment]
have sent the message to their gitter to confirm the files should be hosted.

**Comment by maruilian11** [comment]
@cdnjs/library-reviewer Please help me review this PR, thanks!

File: ajax/libs/mint-ui/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "mint-ui",
+  "version": "2.2.7",
+  "filename": "mint-ui.common.min.js",
+  "description": "Mobile UI elements for vue.js",
+  "keywords": [
+    "component",
+    "vue",
+    "mobile",
+    "eleme"
```
Comment (sufuf3) [inline ajax/libs/mint-ui/package.json]: Maybe we can add more keywords, such as `ui-kit`, which under the GitHub repo's description blue tag part.
What do you think?

Comment (sufuf3) [inline ajax/libs/mint-ui/package.json]: Maybe we can add more keywords, such as `ui-kit`, which under the GitHub repo's description blue tag part.
What do you think?

**Comment by maruilian11** [comment]
@cdnjs/library-reviewer @sufuf3 Updated the keywords and title.

**Comment by sufuf3** [comment]
@extend1994 @pvnr0082t 
Please help review this PR, Thank you.

File: ajax/libs/mint-ui/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "mobile",
+    "eleme",
+    "vue-components",
+    "ui-kit"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ElemeFE/mint-ui.git"
+  },
+  "homepage": "https://mint-ui.github.io/docs/#/",
```
Comment (extend1994) [inline ajax/libs/mint-ui/package.json]: Should this be just `http://mint-ui.github.io`?

**Comment by maruilian11** [comment]
@extend1994 Update the homepage info.

**Comment by sufuf3** [comment]
@maruilian11 The PR title maybe need to be changed as `[New] Add mint-ui@2.2.7 w/ npm auto-update`. What do you think?
Thank you.

**Comment by PeterDaveHello** [comment]
So is there any reference about the auto-update pattern?

**Comment by maruilian11** [comment]
@PeterDaveHello I decided the pattern according to the document. http://mint-ui.github.io/docs/#/en
And `mint-ui.common.js` is also put into the same directory.

**Comment by PeterDaveHello** [comment]
Why did you add map file for only `mint-ui.common.js`?

**Comment by maruilian11** [comment]
It seems like a mistake, sorry about that.

**Pull request merged**